src/core/CorePool.ts

```diff
@@ -100,10 +101,29 @@ export class CorePool {
     tickUpper: number,
     amount: JSBI
   ): { amount0: JSBI; amount1: JSBI } {
-    // TODO
+    if (JSBI.lessThanOrEqual(amount, ZERO)) {
+      console.error("[Error]: mint amount should greater than 0");
+    }
+
+    let amount0 = ZERO;
+    let amount1 = ZERO;
+
+    if (JSBI.greaterThan(amount, ZERO)) {
+      let positionStep = this.modifyPosition(
```

Comment:
Will it be better if just name return object as `position`, or does `step` here stand for something special?

Reply:
In my understand, the returned obj contains `{position, amount0, amount1}`, so here I use a name as step to seperate it's an obj.

Reply:
I see it's ok! For this case, destructuring assignment is also proper to be used.
Here is a case in swap()
https://github.com/Bella-DeFinTech/uniswap-v3-simulator/blob/9e4f5a468fac0dd7849570124f78d1118858a99d/src/core/CorePool.ts#L201

Reply:
That's cool, I'll use destructuring assignment to impl this part, nice review!